ehrapy/anndata/_feature_specifications.py

```diff
@@ -0,0 +1,97 @@
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from rich import print
+from rich.tree import Tree
+
+from ehrapy import logging as logg
+from ehrapy.anndata._constants import CATEGORICAL_TAG, CONTINUOUS_TAG, DATE_TAG, FEATURE_TYPE_KEY
+from ehrapy.anndata.anndata_ext import anndata_to_df
+
+
+def infer_feature_types(adata, layer: str | None = None, output: Literal["print", "dataframe"] | None = "print"):
+    """
+    Infer feature types from AnnData object.
+
+    For each feature in adata.var_names, the method infers one of the following types: 'date', 'categorical', or 'continuous'.
+    The inferred types are stored in adata.var['feature_type']. Please check the inferred types and adjust if necessary using
+    adata.var['feature_type']['feature1']='corrected_type'.
+
+    Args:
+        adata: :class:`~anndata.AnnData` object storing the EHR data.
+        layer: The layer to use from the AnnData object. If None, the X layer is used.
+        output: The output format. Choose between 'print', 'dataframe', or None. If 'print', the feature types will be printed to the console.
```

Comment:
`print` is fine, but maybe `console` or `stdout` would be better? Thoughts?

Reply:
Yes, I also think `print` is not ideal. Out of the two you suggested, I'd go for `console`. Or maybe even something incorporating "tree"? `print_tree`?

Reply:
Just `tree`? Good idea